docs/how-to/redeploy.md

```diff
@@ -0,0 +1,19 @@
+# How to redeploy Jenkins
+
+We consider a redeployment is the process where the old charm instance is removed and data is migrated to a new charm instance.
+
+As Jenkins does not use any persistent storage or database, redeploying the charm consists of 3 steps:
```

Comment:
Don't we have a volume that we attach for configuration, plugins etc which is persistent storage?

Reply:
Yes, I put `persistent` as a mistake, I'll change it to "As Jenkins does not use a database"

Reply:
In the end I thought the first part is not necessary, so I removed it entirely.